pkg/operator/controller/add.go

```diff
@@ -83,7 +83,15 @@ func AddToManager(mgr manager.Manager, cfg *config.OperatorConfiguration) error
 			return err
 		}
 
-		if err := (&service.Reconciler{}).AddToManager(mgr, predicate.Or(virtualGardenKubeAPIServerPredicate, virtualGardenIstioIngressPredicate)); err != nil {
+		nginxIngressPredicate, err := predicate.LabelSelectorPredicate(metav1.LabelSelector{MatchLabels: map[string]string{
+			"app":       "nginx-ingress",
+			"component": "controller",
+		}})
+		if err != nil {
+			return err
+		}
+
+		if err := (&service.Reconciler{}).AddToManager(mgr, predicate.Or(virtualGardenKubeAPIServerPredicate, virtualGardenIstioIngressPredicate, nginxIngressPredicate)); err != nil {
```

Comment:
https://github.com/gardener/gardener/blob/2514f16f8b48696ba21c9275ba3f5ba688fdbfa7/example/gardener-local/kind/operator/values.yaml#L4-L5 controls whether the local port mapping is established. I think we need this if we want to access the `nginx-ingress` locally. Perhaps it should be renamed since it's no longer specific to the seed only now?